documentation/docs/extending/camera-manager.mdx

```diff
@@ -0,0 +1,185 @@
+---
+id: camera-manager
+title: Custom CameraManager
+description: Shows how to implement custom CameraManager to enable new camera manipulation features.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+By default `Cognite3DViewer` uses `DefaultCameraManager` class to manage user interaction with the camera.
+In certain cases it could be useful to customize behaviour of the camera:
+
+- Need to provide custom `THREE.PerspectiveCamera` object to `Cognite3DViewer`
+- Need to extend camera controls to any specific case that is not covered by current implementation
+
+
+## Overview
+
+To create a custom camera manager class, `CameraManager` interface from `@cognite/reveal` must be implemented
+and provided to `Cognite3DViewer` on construction using the `cameraManager`-option. Interface looks like this:
+
+```js
+export interface CameraManager {
```

Comment:
Sure, I pasted it because I remember we can't link to reference. Probably a good idea is to finally update docusaurus and check whether it will fix it.

Reply:
So, I think we can leave it as is and have a reference after we updated docusaurus.